git.lua

```diff
@@ -258,15 +264,29 @@ local stashes = function(token)  -- luacheck: no unused args
     -- generate matches and match filter table
     local ret = {}
     local ret_filter = {}
+    local clink_version_encoded = clink.version_encoded or 0
     for i,v in ipairs(stash_times) do
-        table.insert(ret, "stash@{"..(i-1).."}")
-        table.insert(ret_filter, "stash@{"..(i-1).."}    "..stashes[v])
+        local match = "stash@{"..(i-1).."}"
+        table.insert(ret, match)
+        if clink_version_encoded >= 10010012 then
+            table.insert(ret_filter, { match=match, type="word", description=stashes[v] })
+        elseif clink_version_encoded >= 10010009 then
+            table.insert(ret_filter, "stash@{"..(i-1).."}\x1b[m    "..stashes[v])
```

Comment:
You're resetting color to default here (`\x1b[m` sequence is about that IIRC) but I can't see where the color been chnaged from default

Reply:
Clink itself can set the color (per the `color.filtered` setting).  The reset here forces normal text for the description.  The effect is the stash name in a user-defined color (default is "bold") and the stash description in another color, the default color.  So the default effect is bright text for the name and dim text for the description.

I can add a comment to make it clear.

Reply:
I took the simpler approach of only removing the support for v1.1.9 through v1.1.11 since they're testing builds and v1.1.12 has further enhancements.  The version checks are encapsulated in the version module.